api/apps/graphql/resolvers.js

```diff
@@ -257,6 +257,25 @@ export const relatedArticlesCanvas = (root) => {
   })
 }
 
+export const seriesArticle = (root) => {
+  return new Promise(async (resolve, reject) => {
```

Comment:
Does this mean we loop through every series article to find if it includes a child?

Reply:
Yep, that's how we have to find super articles too. Thankfully there's just one for now, and the real optimization we can make is the `is_series_sub_article` (which we do for super currently). On the Force side, I'm thinking to only make this fetch for video layouts until we have a `is_series_sub_article` ready.

Reply:
is `is_series_sub_article` planned to be a bool or an id?

Reply:
Mmm I suppose it could be either --

But now that I'm thinking again, the `is_series_sub_article` as a bool or id isn't actually an optimization. Because in order to get that, we still need to loop through all `series` articles to find the `id`. So really the best option is to add the series article as `series_article_id` on the sub article and fetch it that way. But as we've discussed, this involves propagating updates to multiple articles within the Series article which we don't have time for.